notebooks/238-typo-detector/238-typo-detector.ipynb

```diff
@@ -0,0 +1,583 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install optimum[openvino]
</pre><p>please use -q for installing dependencies</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done